docs/proposals/22-autoscaling-kube-apiserver-via-independent-hpa-and-vpa.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Autoscaling Shoot `kube-apiserver` via Independently Driven HPA and VPA
```

Comment:
The document should be linked in the docs index

Reply:
This is not addressed

Reply:
Bumped document number to 23 and added to index.